custom/data.ts

```diff
@@ -1,73 +1,72 @@
-import { faker } from "@faker-js/faker";
```

Comment:
Consider organizing imports for better readability and maintainability.

For reference, see [ESLint import/order rule](https://eslint.org/docs/rules/import-order).